universal-application-tool-0.0.1/app/services/applicant/ApplicantData.java

```diff
@@ -16,10 +17,22 @@ public void put(String path, String key, String value) {
     this.jsonData.put(path, key, value);
   }
 
+  public void put(String path, String key, Object value) {
```

Comment:
issue (blocking): Remove this since it's no longer used, and since we're deprecating the other `put` anyway.